csm/core/controllers/information.py

```diff
@@ -77,3 +77,165 @@ async def post(self):
             Log.error(f"Error in checking compatability: {e}")
             raise CsmInternalError(f"{e}")
         return CsmResponse(response)
+
+@CsmAuth.public
+@CsmView._app_routes.view("/api/v2/system/topology")
+class TopologyView(CsmView):
+    """
+    GET: Complete deployment topology
+    """
+    def __init__(self, request):
+        super().__init__(request)
+        self._service = self.request.app[const.INFORMATION_SERVICE]
+
+    async def get(self):
+        """GET REST implementation for complete topology."""
+        Log.info(f"Processing request: {self.request.method} {self.request.path}")
```

Comment:
In log.info, user info is missing in all the controllers.

Reply:
Updated